Automattic/studio
# Studio: fix site not found error when add a site

Fixes https://github.com/Automattic/dotcom-forge/issues/8624

## Proposed Changes

When we add a new site, the following error is thrown `'executeWPCLiInline': Error: Site not found`.
This PR fixes that.

## Testing Instructions


1. Create a new site
2. Ensure that there is no error in the console

## Pre-merge Checklist
